pinot-common/src/main/java/com/linkedin/pinot/common/utils/ZkUtils.java

```diff
@@ -31,37 +29,23 @@
  */
 public class ZkUtils {
   /**
-   * Returns a handle to the propertyStore, setting up a watch on all nodes under PROPERTYSTORE/clusterName, if
-   * setWatch is specified as true.
+   * Returns a handle to the propertyStore
    *
    * @param helixManager * A valid helix manager bound to a cluster.
    * @param clusterName The pathname under propertyStore that we want a handle for
-   * @param setWatch Must be set to true if it is desired to set up a watch on all nodes under the PROPERTYSTORE node.
    * @return A handle to the propertyStore.
    *
    * @note: Setting up a watch can lead to connection problems with zookeeper during a re-connect from the client. The
    * client ends up sending all the path names in the session request, and that can exceed the max size of a session
    * request if there are too many nodes, causing the connection to be dropped and retried (with the same parameters).
    */
-  public static ZkHelixPropertyStore<ZNRecord> getZkPropertyStore(HelixManager helixManager, String clusterName, boolean setWatch) {
+  public static ZkHelixPropertyStore<ZNRecord> getZkPropertyStore(HelixManager helixManager, String clusterName) {
```

Comment:
The method was being used in one place, called with 'false'. I removed it in favor of the other method that is always defaulting to false. We should not be setting a watch on all of propertystore anytime, so we should not even need that method.